### ISSUE TITLE
ELRI local server: check on it

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#2682]
I wanted to patch ELRI local and to check the QR code size and new registration QR codes. Could not connect to kiosk. I went to look at the local server and it tells me it is repairing disk errors and might take an hour, with a spinning thing and the Windows icon. Looks very dangerous and I walked away. I hope you made that happen intentionally.

**Comment by urapadmin** [comment]
no, Windows made that happen and apparently it is not kidding about the estimated time that takes. Things that never happen in the field, it seems ...
